Service Defaults Clarification (#485)
microsoft/aspire.dev: src/frontend/src/content/docs/fundamentals/service-defaults.mdx

Cloud-native applications often require extensive configurations to ensure they work across different environments reliably and securely. Aspire provides many helper methods and tools to streamline the management of configurations for OpenTelemetry, health checks, environment variables, and more.

> File-based Apphost configurations do not come with the ServiceDefaults project automatically.  If you are using a file-based apphost, you can run the following command to generate one for your resources that need it:

Reviewer: ```suggestion
File-based AppHost configurations don't include the [Service defaults project](/fundamentals/service-defaults/). When using a file-based AppHost, run the following command to generate one for the resources that need it:
```
